docs/api/web-contents.md

```diff
@@ -130,10 +130,6 @@ Corresponds to the points in time when the spinner of the tab stopped spinning.
 
 #### Event: 'dom-ready'
 
-Returns:
-
-* `event` Event
-
```

Comment:
Is this removal intentional?

It creates this artifact diff:

```
-    on(event: 'dom-ready', listener: (event: Event) => void): this;
-    once(event: 'dom-ready', listener: (event: Event) => void): this;
-    addListener(event: 'dom-ready', listener: (event: Event) => void): this;
-    removeListener(event: 'dom-ready', listener: (event: Event) => void): this;
+    on(event: 'dom-ready', listener: Function): this;
+    once(event: 'dom-ready', listener: Function): this;
+    addListener(event: 'dom-ready', listener: Function): this;
+    removeListener(event: 'dom-ready', listener: Function): this;
```

...which is maybe a good change since AFAICT there's nothing the user can do with `event`, but as far as docs go it does lose the type info about the return type.

Reply:
The return type doesn't matter either. Arguably that's a bug in the ts generator but I don't think it's worth worrying about.

This removal is intentional (though unrelated to the main change). I think the artifact change is a good one.

The Event object will still be passed, though as you noticed it's completely useless, so I think it's correct to drop it from the docs.